packages/samples/headless-ssr/cypress/e2e/ssr.cy.ts

```diff
@@ -56,12 +56,29 @@ describe('headless ssr example', () => {
     cy.vitals(VITALS_THRESHOLD);
   });
 
-  it('should not log any error nor warning', () => {
-    spyOnConsole();
-    cy.visit('/');
-    waitForHydration();
-    cy.wait(1000);
-    cy.get(ConsoleAliases.error).should('not.be.called');
-    cy.get(ConsoleAliases.warn).should('not.be.called');
+  describe('after hydration', () => {
+    beforeEach(() => {
+      spyOnConsole();
+      cy.visit('/');
+      waitForHydration();
+    });
+
+    it('should not log any error nor warning', () => {
+      cy.wait(1000);
+      cy.get(ConsoleAliases.error).should('not.be.called');
+      cy.get(ConsoleAliases.warn).should('not.be.called');
+    });
+
+    it('after submitting a query, should change search results', () => {
+      const getResultTitles = () => cy.get('.result-list li').invoke('text');
```

Comment:
Is this equivalent to the `textContent` DOM query selector?

Reply:
this is equivalent to [jQuery's `.text()`](https://api.jquery.com/text/#text), so it's at-least a close equivalent.